crypto/x509v3/v3_addr.c

```diff
@@ -1195,7 +1203,11 @@ static int addr_validate_path_internal(X509_STORE_CTX *ctx,
     } else {
         i = 0;
         x = sk_X509_value(chain, i);
-        OPENSSL_assert(x != NULL);
+        if (!ossl_assert(x != NULL)) {
+            if (ctx != NULL)
```

Comment:
Though it can't be NULL in this line, because it has to be non-NULL if ext is NULL.